system-variables.md

```diff
@@ -1513,14 +1513,11 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_enable_plan_replayer_capture`
```

Comment:
@songrijie @Yisaer Can users change this system variable and use the related feature (`PLAN REPLAYER CAPTURE`) on **TiDB Cloud**? If yes, which tier?
If not, we need to mark this as not applicable to Cloud.

Reply:
they can use this feature in Cloud